src/style.css

```diff
@@ -40,3 +40,11 @@ footer {
 .rsg--logo-5 h1 {
   display: none;
 }
+
+.grab {
```

Comment:
unfortunately tachyons does not include this, but it probably makes more sense to set this inside the component as inline style, do you agree?

Reply:
I wanted to use it like:
```
    const hoveredHoverable = hovered && target
    const draggingStyle = draggingClasses || 'grabbing o-40 bg-white'
    const hoverStyle = hoveredClasses || 'bg-silver'

    const classes = classNames(className, 'ba pt2 pb2 pl3 pr3 truncate br4 grab', {
      [draggingStyle]: dragging,
      [hoverStyle]: hoveredHoverable,
    })
```

Reply:
it makes sense to have a draggingClasses prop but the cursor grab could be set as a default behaviour in my opinion, so draggingClasses could be used to extend with other properties, but the grab always shows up